app/components/markdown/markdown.tsx

```diff
@@ -352,16 +352,18 @@ const Markdown = ({
         return rendered;
     };
 
-    const renderImage = ({linkDestination, context, src, size}: MarkdownImageRenderer) => {
+    const renderImage = ({linkDestination, context, src, size, isInsideLink = false}: MarkdownImageRenderer) => {
         if (!imagesMetadata || isUnsafeLinksPost) {
             return null;
         }
 
+        const isImageDisabled = (disableGallery ?? Boolean(!location)) || isInsideLink;
```

Comment:
nit: I was wondering if you think isImageDisabled is somewhat confusing?  I thought when I read `isImageDisabled`, it means the image will not going to be displayed.

Reply:
that makes sense.

Renamed to `disableInteraction`, what do you think?